SUI/SUI/ContentView.swift

```diff
@@ -7,18 +7,166 @@
+        Image(playerView.getTrackImage())
+            .resizable()
+            .frame(width: CGFloat(Constants.trackImageWidth),
+                   height: CGFloat(Constants.trackImageHeight))
+            .scaledToFit()
+            .cornerRadius(CGFloat(Constants.trackImageCornerRadius))
+    }
+    
+    private var setupDurationSlider: some View {
+         Slider(value: Binding(get: {
+            Double(playerView.playerCurrentTime ?? Float(Constants.floatDefaultValue))
+        }, set: { newValue in
+            playerView.playerCurrentTime = Float(newValue)
+            playerView.setTime(value: Float(playerView.playerCurrentTime ?? Float(Constants.floatDefaultValue)))
+        }), in: 0...Double(playerView.maxDuration)) {
+        } minimumValueLabel: {
+            Text(Constants.sliderMinimumValueText)
+        } maximumValueLabel: {
+            Text("\((playerView.maxDuration % Constants.secondsInHour) / Constants.secondsInMinute):\((playerView.maxDuration % Constants.secondsInHour) % Constants.secondsInMinute)")
         }
         .padding()
     }
+    
+    private var setupSongNameLabel: some View {
+        Text(playerView.getTrackName())
+    }
+    
+    private var setupPlayAndStopButtons: some View {
+          HStack {
+            Button {
+                playerView.play(trackName: playerView.getTrackName())
+            } label: {
+                Image(systemName: Constants.playImageName)
+                    .frame(width: CGFloat(Constants.buttonImageWidth),
+                           height: CGFloat(Constants.buttonImageHeight))
+                    .foregroundColor(.white)
+            }.background(.orange)
+                .cornerRadius(CGFloat(Constants.buttonImageCornerRadius))
+            
+            Button {
+                playerView.stop()
+                progress = Float(Constants.floatDefaultValue)
+            } label: {
+                Image(systemName: Constants.stopImageName)
+                    .frame(width: CGFloat(Constants.buttonImageWidth),
+                           height: CGFloat(Constants.buttonImageHeight))
+                    .foregroundColor(.white)
+            }.background(.orange)
+                .cornerRadius(CGFloat(Constants.buttonImageCornerRadius))
+            
+        }
+    }
+    
+    private var setupNextAndPreviosTrackButtons: some View {
+         HStack {
+            Button {                playerView.nextTrack()
+                playerView.setTime(value: Float(Constants.floatDefaultValue))
+            } label: {
+                Image(systemName: Constants.backwardImageName)
+                    .frame(width: CGFloat(Constants.buttonImageWidth),
+                           height: CGFloat(Constants.buttonImageHeight))
+                    .foregroundColor(.white)
+            }.background(.orange)
+                .cornerRadius(CGFloat(Constants.buttonImageCornerRadius))
+            
+            Button {
+                playerView.previousTrack()
```

Comment:
```suggestion
                playerView.previousTrack()
                playerView.setTime(value: Float(Constants.floatDefaultValue))
```
это дожно вызывается из vm